packages/analytics-browser/src/browser-client.ts

```diff
@@ -80,6 +83,11 @@ export class AmplitudeBrowser extends AmplitudeCore<BrowserConfig> {
 
     // Step 6: Track attributions
     await this.runAttributionStrategy(browserOptions.attribution, isNewSession);
+
+    // Step 7:
+    // Run queued functions
+    this.q = [...delayedQ, ...this.q];
+    await this.runQueuedFunctions();
```

Comment:
do we need `delayedQ` separated out?

Reply:
Pulling these items off the queue before init() allows the plugin queue to run in core _init() while waiting until other initialization is complete for AmplitudeBrowser.